### PR TITLE
Bug sitedetails

### PR DESCRIPTION
**Pull request by ephetic**

### CODE REVIEW

File: app/components/TourSection.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Modal from 'react-modal';
 import FilterSection from './FilterSection';
 import SearchSection from './SearchSection';
 import {getLocation} from '../utils/geo';
-import TourList from './TourList';
+// import TourList from './TourList';
```
Comment (ha404) [inline app/components/TourSection.jsx:8]: can we remove this?

File: app/components/TourSection.jsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default class TourSection extends React.Component {
   }
 
   render() {
-    const styles = this.props.params.tourId ? 'TourSection__TourList--hidden' : 'TourSection__TourList';
+    // const styles = this.props.params.tourId ? 'TourSection__TourList--hidden' : 'TourSection__TourList';
```
Comment (ha404) [inline app/components/TourSection.jsx]: do we still need this?

File: app/components/TourSection.jsx [hunk 3]
```diff
@@ -103,6 +103,7 @@ export default class TourSection extends React.Component {
           {...this.state}
           {...this.props}
         />
+        {/*
```
Comment (ha404) [inline app/components/TourSection.jsx]: do we need this anymore?

File: app/components/TourSection.jsx (hunk 2 above)
Comment (ephetic) [inline app/components/TourSection.jsx]: no

File: app/components/TourSection.jsx (hunk 3 above)
Comment (ephetic) [inline app/components/TourSection.jsx]: it's a hedge against reverting this PR

**Pull request merged**